[2024-04-28T19:06:58.242+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-28T19:06:58.266+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_yellow_cab.validate_file manual__2024-04-28T19:06:22.575085+00:00 [queued]>
[2024-04-28T19:06:58.275+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_yellow_cab.validate_file manual__2024-04-28T19:06:22.575085+00:00 [queued]>
[2024-04-28T19:06:58.275+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-28T19:06:58.290+0000] {taskinstance.py:2327} INFO - Executing <Task(BranchPythonOperator): validate_file> on 2024-04-28 19:06:22.575085+00:00
[2024-04-28T19:06:58.295+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=312) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-28T19:06:58.297+0000] {standard_task_runner.py:63} INFO - Started process 313 to run task
[2024-04-28T19:06:58.297+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_yellow_cab', 'validate_file', 'manual__2024-04-28T19:06:22.575085+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_cab.py', '--cfg-path', '/tmp/tmptvmm1r3h']
[2024-04-28T19:06:58.300+0000] {standard_task_runner.py:91} INFO - Job 51: Subtask validate_file
[2024-04-28T19:06:58.346+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_yellow_cab.validate_file manual__2024-04-28T19:06:22.575085+00:00 [running]> on host 3b50846985b8
[2024-04-28T19:06:58.439+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_yellow_cab' AIRFLOW_CTX_TASK_ID='validate_file' AIRFLOW_CTX_EXECUTION_DATE='2024-04-28T19:06:22.575085+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-28T19:06:22.575085+00:00'
[2024-04-28T19:06:58.440+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-28T19:06:58.454+0000] {baseoperator.py:399} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-04-28T19:06:58.458+0000] {_universal.py:514} INFO - Request URL: 'https://azureyellowcab.blob.core.windows.net/cabfiles?restype=REDACTED&comp=REDACTED'
Request method: 'GET'
Request headers:
    'x-ms-version': 'REDACTED'
    'Accept': 'application/xml'
    'User-Agent': 'azsdk-python-storage-blob/12.19.1 Python/3.12.2 (Linux-6.6.22-linuxkit-x86_64-with-glibc2.36)'
    'x-ms-date': 'REDACTED'
    'x-ms-client-request-id': '7c8a4a44-0592-11ef-8aca-0242c0a81004'
    'Authorization': 'REDACTED'
No body was attached to the request
[2024-04-28T19:06:58.797+0000] {_universal.py:550} INFO - Response status: 200
Response headers:
    'Transfer-Encoding': 'chunked'
    'Content-Type': 'application/xml'
    'Server': 'Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0'
    'x-ms-request-id': 'df8eab5f-101e-0055-0b9f-99d814000000'
    'x-ms-client-request-id': '7c8a4a44-0592-11ef-8aca-0242c0a81004'
    'x-ms-version': 'REDACTED'
    'Date': 'Sun, 28 Apr 2024 19:06:58 GMT'
[2024-04-28T19:06:58.819+0000] {logging_mixin.py:188} INFO - CertificateReport.pdf
[2024-04-28T19:06:58.820+0000] {logging_mixin.py:188} INFO - yellow_tripdata_2011-01.parquet
[2024-04-28T19:06:58.821+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-28T19:06:58.822+0000] {branch.py:36} INFO - Branch into None
[2024-04-28T19:06:58.823+0000] {skipmixin.py:178} INFO - Following branch None
[2024-04-28T19:06:58.831+0000] {skipmixin.py:246} INFO - Skipping tasks [('file_exception', -1)]
[2024-04-28T19:06:58.853+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-28T19:06:58.861+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_yellow_cab, task_id=validate_file, execution_date=20240428T190622, start_date=20240428T190658, end_date=20240428T190658
[2024-04-28T19:06:58.887+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-28T19:06:58.904+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-28T19:06:58.906+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
